scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -288,7 +288,7 @@
           <span class="float-end">{{ variant.spidex_human }}</span>
         </li>
         <li class="list-group-item">
-          <a href="https://doi.org/10.1016/j.cell.2018.12.015" target="_blank">SpliceAI</a> <a href="https://github.com/Illumina/SpliceAI" target="_blank">DS max</a>
+          <a href="https://doi.org/10.1016/j.cell.2018.12.015" target="_blank">SpliceAI</a> (<a href="{{ variant.spliceai_link }}">Lookup</a>) <a href="https://github.com/Illumina/SpliceAI" target="_blank">DS max</a>
```

Comment:
```suggestion
          SpliceAI (<a href="{{ variant.spliceai_link }}" target="_blank" rel="noopener">Lookup</a>) <a href="https://github.com/Illumina/SpliceAI" target="_blank" rel="noopener">DS max</a>
```

Reply:
Actually, we could just switch the links then and skip the lookup.. 😅